### DISCUSSION TITLE
Question: Webhook

### DISCUSSION THREAD

**Discussion by poudenes** [prodrigestivill/docker-postgres-backup-local#127] [Q&A]
Hi All,

I'm using this tool with love. Want to add web hook option into it.
When I use this and try readout the JSON output I only see:

```
{
  "status": "post-backup"
}
```

How can I add more information to it? Like Backup successful or there is an error etc?

**Comment by prodrigestivill** [comment]
Right now the status can be, "post-backup" or "error", depending on how it finished.
Also "pre-backup" is also send just before to perform it.

**Reply by arathunku** [reply]
Hello @prodrigestivill :wave: 

Would you be open to PR which customizes that or skipping the data? I'm trying to set up https://docs.sentry.io/product/crons/getting-started/http/, but there's no option to overwrite the `data` param. 

```
# works
$ curl --request POST \
              --url "$SENTRY_CRONS" \
              --header 'Content-Type: application/json' \
              --max-time 10 \
              --retry 5 \
               --data-raw '{"status": "ok"}' 

# doesn't work
$ curl --request POST \
              --url "$SENTRY_CRONS" \
              --header 'Content-Type: application/json' \
              --data '{"status": "post-backup"}'
              --max-time 10 \
              --retry 5 \
               --data-raw '{"status": "ok"}' 
```

**Reply by prodrigestivill** [reply]
You can add your custom hook, similar to [the existing one](https://github.com/prodrigestivill/docker-postgres-backup-local/blob/main/hooks/00-webhook).

To add you hooks, you can mount a custom volume or a K8s _ConfigMap_ at `/hooks`.

Alternatively you can extend the docker by building an image like this:

```Dockerfile
FROM prodrigestivill/postgres-backup-local:17
COPY 00-custom-hook /hooks/00-custom-hook
RUN chmod a+x /hooks/*
```
